src/components/POLine/POLineVersionView/VendorVersionView/VendorVersionView.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  KeyValue,
+  Row,
+} from '@folio/stripes/components';
+import { VendorReferenceNumbersDetails } from '@folio/stripes-acq-components';
+
+import { VersionKeyValue } from '../../../../common/VersionView';
+
+export const VendorVersionView = ({ version }) => {
+  const vendorDetail = version?.vendorDetail;
+
+  return (
+    <>
```

Comment:
Could you please investigate do we need this tag <> at all?

Reply:
react element removed as redundant